src/layman/layer/micka/csw.py

```diff
@@ -183,8 +182,7 @@ def get_template_path_and_values(publication: Layer, *, http_method):
         'temporal_extent': temporal_extent,
         'wms_url': f"{wms.add_capabilities_params_to_url(wms.get_wms_url(external_url=True))}&LAYERS={wms_name}",
         'wfs_url': f"{wfs.add_capabilities_params_to_url(wfs_url)}&LAYERS={wfs_name}" if wfs_url else None,
-        'layer_endpoint': url_for('rest_workspace_layer.get', workspace=publication.workspace,
-                                  layername=publication.name),
+        'layer_endpoint': url_for('rest_layer.get', uuid=publication.uuid),
```

Comment:
Subsequent PR will be needed for metadata migration of identifier & layer_endpoint.